extensions/binding/org.eclipse.smarthome.binding.homematic/src/main/java/org/eclipse/smarthome/binding/homematic/internal/type/HomematicTypeGeneratorImpl.java

```diff
@@ -162,8 +163,8 @@ public void generate(HmDevice device) {
                     if (groupType == null || device.isGatewayExtras()) {
                         String groupLabel = String.format("%s",
                                 WordUtils.capitalizeFully(StringUtils.replace(channel.getType(), "_", " ")));
-                        groupType = new ChannelGroupType(groupTypeUID, false, groupLabel, null, null,
-                                channelDefinitions);
+                        groupType = ChannelGroupTypeBuilder.instance(groupTypeUID, groupLabel).isAdvanced(false)
```

Comment:
`isAdvanced(false)` is default for `ChannelGroupTypeBuilder`. We can skip this one.